Tribler/Main/vwxGUI/list_details.py

```diff
@@ -522,20 +525,23 @@ def updateDetailsTab(self):
         self.channel.Show(show_channel)
 
         # Toggle thumbnails
-        # FIXME(lipu): fix the thumbnail path to use metadata
-        thumb_dir = os.path.join(u"",
-                                 binascii.hexlify(self.torrent.infohash))
-        thumb_files = [os.path.join(dp, fn) for dp, _, fns in os.walk(thumb_dir)
-                       for fn in fns if os.path.splitext(fn)[1] in THUMBNAIL_FILETYPES]
-        show_thumbnails = bool(thumb_files)
+        if self.torrent.metadata and 'thumb_hash' in self.torrent.metadata:
```

Comment:
@NielsZeilemaker I found out why. There is a bug in when loading ChannelTorrents on demand: it somehow assigned a wrong channel to this ChannelTorrent. That's why it couldn't fetch its metadata.